cli/js/build.ts

```diff
@@ -1,24 +1,25 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-export type OperatingSystem = "mac" | "win" | "linux";
+//@ts-ignore
```

Comment:
I know you hate TS getting in the way, but why just not do:

```ts
export const build = {
  target: "unknown",
  arch: "unknown",
  os: "unknown",
  vendor: "unknown"
  env: undefined as (string | undefined)
};
```

Heck of a lot better than just ignoring errors.

Reply:
Here is a [working example](https://www.typescriptlang.org/play/#code/KYDwDg9gTgLgBAYwgOwM7wEYFcCWAbAEzgF44BvAKDjhgEMoBzYGALjgCItkBrZCAd2TsANFTj0EACzacefQSLERUMrrwFDR1AG7BkBaKrkbF1PdrZcCwAGY5kwIrVRwAFOij2GcAD5wrtvaOAJQUAL4A3BQUoJCwcDZcCDA4KHCozABCuIQAksg2EK50jMxsHl7BbNoQOESU1NRIaPAA2hKSwnC6+tBdyl3mALokNPRMMAB0qGB4ODCu7AC0InAALMFRjXDY+ASTJROjh8xbjbuEkx2jHWfUF-s9BlCjT9B3Ozn7yqPKHw+TcyjcxRMIUIA) in strict mode with no ignoring of errors and no real "trickery" except for the env.

Reply:
Thanks - fixed.